fatiando/gravmag/harvester.py

```diff
@@ -538,8 +539,6 @@ def _neighbor_indexes(n, mesh, restrict):
     """Find the indexes of the neighbors of n"""
     nz, ny, nx = mesh.shape
     indexes = []
-    if restrict is None:
-        restrict = []
```

Comment:
With _test_restriction() returning an empty list, this should be superfluous now.